.github/workflows/build.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build ARM user
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install deps
+      run: |
+        apt --no-install-recommends -y build-dep qemu
```

Comment:
we should use `apt-get` in actions as `apt` has an unstable interface, also don't forget to use `sudo` here.